tests/acceptance/features/apiSharingNg/updateShareInvitations.feature

```diff
@@ -0,0 +1,156 @@
+Feature: Update permission of a share
+  https://owncloud.dev/libre-graph-api/#/drives.permissions/UpdatePermission
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario: user updates expiration date of a share
+    Given user "Alice" has uploaded file with content "hello world" to "testfile.txt"
+    And user "Alice" has sent the following share invitation:
+      | resource        | testfile.txt |
+      | space           | Personal     |
+      | sharee          | Brian        |
+      | shareType       | user         |
+      | permissionsRole | Viewer       |
+    When user "Alice" updates the last share with the following using the Graph API:
+      | space              | Personal             |
+      | resource           | testfile.txt         |
+      | expirationDateTime | 2200-07-15T14:00:00Z |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "expirationDateTime",
+        "grantedToV2",
+        "id",
+        "roles"
+      ],
+      "properties": {
+        "expirationDateTime": {
+          "type": "string",
+          "enum": ["2200-07-15T14:00:00Z"]
+        },
+        "grantedToV2": {
+          "type": "object",
+          "required": [
+            "user"
+          ],
+          "properties":{
+            "user": {
+              "type": "object",
+              "required": [
+                "displayName",
+                "id"
+              ],
+              "properties": {
+                "displayName": {
+                  "type": "string",
+                  "enum": ["Brian Murphy"]
+                },
+                "id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                }
+              }
+            }
+          }
+        },
+        "id": {
+          "type": "string",
+          "pattern": "^%permissions_id_pattern%$"
+        },
+        "roles": {
+          "type": "array",
+          "items": [
+            {
+              "type": "string",
+              "pattern": "^%role_id_pattern%$"
+            }
+          ]
+        }
+      }
+    }
+    """
+
+
+  Scenario Outline: user updates role of a share
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has sent the following share invitation:
+      | resource        | <path>             |
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
```

Comment:
```suggestion
      | permissionsRole | <previous-permissions-role> |
```
The `previousPermissionsrole` is used in other share update scenarios, so here you can use `previous-permissions-role`. The `permissions-role` is also fine but in the example table it will be more clear if we use `previous-permissions-role`.